### PR TITLE
2.6.0 upgrading guide

### PR DESCRIPTION
**Pull request by vlad-diachenko**
manually backported upgrading guide from release 2.6.0

### CODE REVIEW

File: docs/sources/upgrading/_index.md [hunk 1]
```diff
@@ -38,6 +38,14 @@ The output is incredibly verbose as it shows the entire internal config struct u
 We now evenly spread queriers across the available kubernetes nodes, but allowing more than one querier to be scheduled into the same node.
 If you want to run at most a single querier per node, set `$._config.querier.use_topology_spread` to false.
 
+#### Default value for `server.http-listen-port` changed
```
Comment (slim-bean) [inline docs/sources/upgrading/_index.md:41]: I think this should be under the Loki section of 2.6.0? looks like it got out of place?

Comment (slim-bean) [reply docs/sources/upgrading/_index.md:41]: oh interesting, this PR moves it, is this because it's not in the 2.6.0 release?

Comment (vlad-diachenko) [reply docs/sources/upgrading/_index.md:41]: Christian added it to the bottom of the section

**Review by slim-bean — APPROVED** [review]
LGTM

**Review by ssncferreira — APPROVED** [review]
LGTM 🏆

**Pull request merged**